R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
I dislike “utils” modules. They are usually a clue that the architecture is amiss.